pdal/util/FileUtils.cpp

```diff
@@ -431,6 +436,7 @@ std::vector<std::string> glob(std::string path)
     if (path[0] == '~')
         throw pdal::pdal_error("PDAL does not support shell expansion");
 
+#ifdef _WIN32
```

Comment:
What is this bit doing?

Reply:
I dropped this part at it was incomplete

Reply:
FTR I did some independent work as part of an attempt to update pdal in vcpkg. For `glob` in mingw, I switch to the A functions:

https://github.com/dg0yt/vcpkg/blob/ce74d9653848718f1044b3a5a10d17a7a8f004e6/ports/pdal/fix-mingw.patch#L104-L149